### PR TITLE
Import and clone commands

### PR DESCRIPTION
**Pull request by Flink**
Implement `import` and `clone` commands.

### CODE REVIEW

File: commands [hunk 1]
```diff
@@ -170,10 +176,14 @@ case "$1" in
 
   $PLUGIN_COMMAND_PREFIX:clone)
     [[ -z $2 ]] && dokku_log_fail "Please specify a name for the service"
+    [[ -z $3 ]] && dokku_log_fail "Please specify a name for the new service"
     verify_service_name "$2"
     SERVICE="$2"
-
-    dokku_log_fail "Not yet implemented"
+    NEW_SERVICE="$3"
```
Comment (josegonzalez) [inline commands]: You need to verify that this service doesn't already exist.

Comment (Flink) [inline commands]: it’s done by `postgres:create` :)

Comment (josegonzalez) [inline commands]: Please write a test specifically for this case :)

Comment (Flink) [inline commands]: ok!

Comment (Flink) [inline commands]: done :)

**Pull request merged**